config/locales/pl.yml

```diff
@@ -228,6 +228,7 @@ pl:
     delete_account_html: Jeżeli chcesz usunąć konto, <a href="%{path}">przejdź tutaj</a>. Otrzymasz prośbę o potwierdzenie.
     didnt_get_confirmation: Nie otrzymałeś instrukcji weryfikacji?
     forgot_password: Nie pamiętasz hasła?
+    invalid_reset_password_token: Link resetowania hasła jest nieważny lub nieprawidłowy. Spróbuj ponownie.
```

Comment:
Yup, it should be something like “Try to get another one” or tip like “Have you already used it?”